packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -719,323 +650,136 @@ class TimeInput extends React.Component {
-        />
-        <span className={cx('time-spacer')}>:</span>
-        <Input
-          {...inputAttributes}
-          {...minuteAttributes}
-          refCallback={(inputRef) => { this.minuteInput = inputRef; }}
-          aria-label={intl.formatMessage({ id: 'Terra.timeInput.minutes' })}
-          className={cx('time-input-minute', showSeconds ? 'with-second' : 'without-second', 'desktop', { 'initial-focus': this.state.minuteInitialFocused })}
-          type="text"
-          value={this.state.minute}
-          name={'terra-time-minute-'.concat(name)}
-          placeholder={intl.formatMessage({ id: 'Terra.timeInput.mm' })}
-          maxLength="2"
-          onChange={this.handleMinuteChange}
-          onKeyDown={this.handleMinuteInputKeyDown}
-          onFocus={this.handleMinuteFocus}
-          onBlur={this.handleMinuteBlur}
-          size="2"
-          pattern="\d*"
-          disabled={disabled}
-        />
-        {showSeconds && (
-          <React.Fragment>
-            <span className={cx('time-spacer')}>:</span>
-            <Input
-              {...inputAttributes}
-              {...secondAttributes}
-              refCallback={(inputRef) => { this.secondInput = inputRef; }}
-              aria-label={intl.formatMessage({ id: 'Terra.timeInput.seconds' })}
-              className={cx('time-input-second', 'desktop', { 'initial-focus': this.state.secondInitialFocused })}
-              type="text"
-              value={this.state.second}
-              name={'terra-time-second-'.concat(name)}
-              placeholder={intl.formatMessage({ id: 'Terra.timeInput.ss' })}
-              maxLength="2"
-              onChange={this.handleSecondChange}
-              onKeyDown={this.handleSecondInputKeyDown}
-              onFocus={this.handleSecondFocus}
-              onBlur={this.handleSecondBlur}
-              size="2"
-              pattern="\d*"
-              disabled={disabled}
-            />
-          </React.Fragment>
-        )}
-        {this.props.variant === TimeUtil.FORMAT_12_HOUR && (
-          [
-            <Input
-              {...inputAttributes}
-              aria-label={intl.formatMessage({ id: 'Terra.timeInput.display.meridiem' })} // value in translations set to 'Display Meridiem'
-              aria-readonly
-              className={cx(['meridiem-display', { focused: this.state.meridiemFocused }])}
-              onFocus={this.handleMeridiemInputFocus}
-              key="meridiem_display"
-              tabIndex="-1"
-              value={this.state.meridiem}
-              size={this.state.meridiem.length || 1}
-              readOnly
-              disabled={disabled}
-            />,
-            <div
-              className={cx('meridiem-select-wrapper')}
-              key="meridiem_select_box"
-            >
-              <select
-                aria-label={intl.formatMessage({ id: 'Terra.timeInput.select.meridiem' })} // value in translations set to 'Select Meridiem'
```

Comment:
The `Terra.timeInput.display.meridiem` and `Terra.timeInput.select.meridiem` [translation strings](https://github.com/cerner/terra-framework/blob/master/packages/terra-time-input/translations/en-US.json#L10-L11) can also be removed if they are no longer needed.

Reply:
Good point! I have [removed](https://github.com/cerner/terra-framework/pull/837/commits/293524d4f0bc16b3cab27fa233240a6337d29d58) them.

Reply:
A new en-CA translation files just for merged today: https://github.com/cerner/terra-framework/blob/master/packages/terra-time-input/translations/en-CA.json#L10-L11
Can you also remove it en-CA?